pyvista/core/utilities/transform.py

```diff
@@ -2359,6 +2414,302 @@ def check_finite(self: Transform) -> bool:  # numpydoc ignore: RT01
     def check_finite(self: Transform, value: bool) -> None:
         self._check_finite = bool(value)
 
+    @property
+    def position(self) -> tuple[float, float, float]:  # numpydoc ignore=RT01
+        """Return the translation component of the current :attr:`matrix`.
+
+        .. versionadded:: 0.47
+
+        See Also
+        --------
+        has_translation, translate, decompose
+
+        Examples
+        --------
+        Compose a translation and get the position.
+
+        >>> import pyvista as pv
+        >>> trans = pv.Transform() + (1, 2, 3)
+        >>> trans.position
```

Comment:
Yep, translation sounds better here. I don't like how this is exposing `GetPosition` through the python property `translation`, but it makes more sense here and the VTK getter/setter should be `GetTranslation` / `SetTranslation`.